homeassistant/components/light/zwave.py

```diff
@@ -244,9 +235,9 @@ def _value_added(self, value):
         # Check for the missing color values
         self._get_color_values()
 
-    def update_properties(self):
+    def update(self):
```

Comment:
I agree and the component ABC should not overwrite the update method. However, the ABC from components will extend the root ABC `Entity` which contains an empty `update` method. Now this itself is not a big problem until someone would change the order of the inheritance. I'm not sure if that would even work so I guess it doesn't matter.